src/ur/ur_driver.cpp

```diff
@@ -223,6 +232,78 @@ bool UrDriver::writeTrajectoryControlMessage(const control::TrajectoryControlMes
   return reverse_interface_->writeTrajectoryControlMessage(trajectory_action, point_number);
 }
 
+bool UrDriver::zeroFTSensor()
+{
+  if (getVersion().major < 5)
+  {
+    std::stringstream ss;
+    ss << "Zeroing the Force-Torque sensor is only available for e-Series robots (Major version >= 5). This robot's "
+          "version is "
+       << getVersion();
+    URCL_LOG_ERROR(ss.str().c_str());
+    return false;
+  }
+  else
+  {
+    if (script_command_interface_->clientConnected())
+    {
+      return script_command_interface_->zeroFTSensor();
+    }
+    else
+    {
+      std::stringstream cmd;
```

Comment:
Maybe a warning output about falling back to a secondary program might be useful here?

Reply:
I took the liberty of implementing this, as I wanted to have this for debugging while testing, anyway. See e86b1f8857690de81f06fda0665941ee3b20f7fe